telemetry/csharp/AwsToolkit.Telemetry.Events.Generator/DefinitionsBuilder.cs

```diff
@@ -27,6 +27,8 @@ public class DefinitionsBuilder
         private readonly CodeMethodReferenceExpression _debugAssert = 
             new CodeMethodReferenceExpression(new CodeTypeReferenceExpression(typeof(System.Diagnostics.Debug)), "Assert");
 
+        private readonly CodeTypeReferenceExpression _debuggerAttached = new CodeTypeReferenceExpression(typeof(System.Diagnostics.Debugger));
```

Comment:
This relates to "Debugger" not the Attached property, consider removing Attached from the name.